src/closure-externs/closure-externs.js

```diff
@@ -194,6 +194,21 @@ var wakaUnknownBefore;
  */
 var MozBlobBuilder;
 
+/**
+ * Some JS libraries make conditional calls to dynCall_xxx function when
+ * callbacks are registered.  For example the exit runtime handler makes
+ * conditional calles to dynCall_v and dynCall_vi, but not all wasm binaryies
+ * will contain these.
+ */
+/**
+ * @suppress {duplicate, undefinedVars}
+ */
+var dynCall_v;
+/**
+ * @suppress {duplicate, undefinedVars}
+ */
+var dynCall_vi;
+
```

Comment:
Do we have a test for what this fixes?

Reply:
This is unfortunately not the right way to do this. Placing these in closure-externs.js file is telling Closure "hey, these two functions will be called from outside code that you cannot see, so please do not minify either of these variables", so in all generated code, `dynCall_v` and `dynCall_vi` will then go unminified. And since this is unconditional, this will occur even if one was not using EXIT_RUNTIME=1! (even though this was supposed to fix an issue with Closure + EXIT_RUNTIME=1). Proper fix in at #12046.